ext/rotoscope/rotoscope.c

```diff
@@ -180,6 +180,7 @@ static void event_hook(VALUE tpval, void *data)
   if (in_fork(config))
   {
     rb_tracepoint_disable(config->tracepoint);
+    config->state = RS_OPEN;
```

Comment:
Ah good point that this won't be hit until we get another trace, which would make the API a bit wonky 👍